vassal-app/src/main/java/VASSAL/build/module/chessclockcontrol/ChessClock.java

```diff
@@ -447,43 +448,80 @@ public boolean setTimerButton() {
 
     boolean oldTocking = tocking;
     tocking = (tenths >= 5);
+    String baseline;
 
     if (doDays) {
       if (doTenths) {
         timerButton.setText(String.format("%s%d:%02d:%02d:%02d.%d", getFormattedButtonText(), days, hours, minutes, seconds, tenths)); //NON-NLS
+        baseline = "0:00:00:00.0"; //NON-NLS
       }
       else if (doSeconds) {
         timerButton.setText(String.format("%s%d:%02d:%02d:%02d", getFormattedButtonText(), days, hours, minutes, seconds)); //NON-NLS
+        baseline = "0:00:00:00.0"; //NON-NLS
```

Comment:
Is having the tenths in the string here a copy-paste mistake?

Reply:
Yes :)